woocommerce/sensei-media-attachments
# Add Hungarian translation

Fixes #50.

This adds Hungarian translation files.

## Testing
1. In _Settings_ > _General_, change the _Site Language_ to Hungarian (i.e. Magyar).
2. Go to _Courses_ > _All Courses_ and select a course.
3. Ensure the _Course Media_ metabox title and buttons are translated.
